### ISSUE TITLE
Wildcard in `TIR-Learner2.4.sh` expands to too many elements and cause an error to cp

### ISSUE THREAD

**Issue by KamilSJaron** [oushujun/EDTA#45]
Hello again :-), I have run into the following error:

```
243: EDTA/bin/TIR-Learner2.4/TIR-Learner2.4.sh: cp: Argument list too long
```

the line that dropped the error message in the script is

```
cp -r $genomeName/$genomeName* temp/
```

where variable `genomeName` is statically assigned to `TIR-Learner` at the very beginning of the script.

The `temp/` directory is now empty, which I am not sure if it's a problem or not.

**Comment by oushujun** [comment]
Hi Kamil,

Thanks for the bug report. Yes sometimes when genomes are too fragmented, the wildcard parameter length will be a problem for cp. `temp/` is used for backup, so the result is fine even cp failed, but checkpointing may have some issue since there is no backup in `/temp`. I have changed cp to rsync to avoid this kind of error. The change will be reflected in the next update. 

Best,
Shujun

**Comment by oushujun** [comment]
This bug should be fixed in the v1.7.3. Please let me know if you still have this issue.
